packages/scandipwa/src/component/ProductCustomizableOption/ProductCustomizableOption.container.js

```diff
@@ -109,21 +110,27 @@ export class ProductCustomizableOptionContainer extends PureComponent {
         return !!isRequiredSelected.length;
     }
 
-    renderOptionLabel(priceType, price, currency) {
+    renderOptionLabel(priceType, price) {
         const {
-            finalPrice: {
-                currency: finalPriceCurrency = '',
-                value = 0
-            } = {}
+            price_range: {
+                minimum_price: {
+                    default_final_price_excl_tax: {
+                        currency: finalPriceCurrency = '',
+                        value = 0
+                    } = {}
+                } = {}
+            } = {},
+            type_id = ''
         } = this.props;
 
-        const finalPrice = formatPrice((value * price) / ONE_HUNDRED_PERCENT, finalPriceCurrency);
+        const finalPriceValue = (type_id === 'bundle') ? value : (value * price);
```

Comment:
Please use constants for string values. You can reuse `BUNDLE` from `src/util/Product`.

Reply:
Also there is no need to wrap these expressions in parentheses.

Reply:
Great tips. Thanks for emphasising those points!